src/fe/fe_map.C

```diff
@@ -1662,7 +1662,11 @@ Point FEMap::inverse_map (const unsigned int dim,
                           const Point & physical_point,
                           const Real tolerance,
                           const bool secure,
-                          const bool extra_checks)
+                          const bool
+#ifdef DEBUG
+                          extra_checks
```

Comment:
This is what your `libmesh_dbg_var()` macro is for, right?

Reply:
Ah, the infamous `#ifndef NDEBUG` strikes again. OK, I guess the current macro could remain as-is and we could add e.g. `libmesh_DEBUG_var()` which only triggers `#ifdef DEBUG`.

Reply:
I like this spelling since it appears to be shouting at you.